src/distilabel/pipeline/_dag.py

```diff
@@ -28,11 +28,13 @@
 
 import networkx as nx
 
+from distilabel.utils.serialization_v2 import _get_class, _Serializable
```

Comment:
Maybe you can move this to `src/distilabel/pipeline/serialization.py` temporarily instead, so as to avoid having conflicts with the previous files (we will remove them all)

Also the new stuff is only under `src/distilabel/pipeline` so that may be clearer, and if you're OK with that I can fill a PR afterwards removing the previous/outdated files 👍🏻

Reply:
Totally. The `src/distilabel/pipeline/serialization.py` is done